src/components/Books.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import InputBook from './InputBook';
+import ListBox from './ListBox';
+import Book from './Book';
+
+class Books extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      book: [],
+      bookTitle: '',
+      bookAuthor: '',
+    };
+  }
+
+  handleSubmit = (e) => {
+    const { bookTitle, bookAuthor, book } = this.state;
+    e.preventDefault();
+    if (bookTitle.trim() && bookAuthor.trim()) {
+      this.setState({
+        book: [...book, { title: bookTitle, author: bookAuthor }],
+        bookAuthor: '',
+        bookTitle: '',
+      });
+    }
+  };
+
+    handleChange = (e) => {
+      const { name, value } = e.target;
+      this.setState({
+        [name]: value,
+      });
+    };
+
+    render() {
+      const {
+        genre, title, author, bookTitle, bookAuthor, book,
+      } = this.state;
```

Comment:
- [ ] [OPTIONAL] This is going to cause a huge issue because you have not declared _genre, title, author,_ in the state of this Books.js component. So restructuring this state will result in those variables rendering as undefined.